addons/fire/functions/fnc_burn.sqf

```diff
@@ -162,8 +162,8 @@ if (_isBurning) exitWith {};
                         [QGVAR(burn), [ace_player, _intensity * (7 / 8), _instigator]] call CBA_fnc_globalEvent;
                     };
                 } else {
-                    if ((ace_player isKindOf "Man") && { !(_unit isEqualTo ace_player) }) then {
-                        private _burnCounter = ace_player getVariable [QGVAR(burnCounter), 0];
+                    if ((ace_player isKindOf "Man") && { !(_unit isEqualTo ace_player) } && isDamageAllowed ace_player) then {
```

Comment:
While changing this, `x isNotEqualTo y` is now a thing that can replace `!(x isEqualTo y)`.

Reply:
Changed all the instances of ``!( x isEqualTo y)`` in the file to ``x isNotEqualTo y``